UM/Qt/Bindings/OperationStackProxy.py

```diff
@@ -6,6 +6,7 @@
 from UM.Application import Application
 
 
+#"OperationStackProxy is deprecated and will be removed in major SDK release"
```

Comment:
Could you add a `deprecated` decorator on the constructor method ? This comment is not very visible...

Reply:
Yes I tried, It doesn't work here somehow. Cura start crashing when this class is called then. I provided the deprecated only for the wrapper, cause it shows there. Here, I added a message so that we remove it later by ourself.

Reply:
Ok, constructors are not normal methods indeed, so that's understandable. As long as warnings show up at some point, that's fine.